src/CMakeLists.txt

```diff
@@ -842,6 +854,18 @@ add_subdirectory(compressor)
 
 add_subdirectory(tools)
 
+# dmClock
+
+add_subdirectory(dmclock) # after gmock
+add_dependencies(tests dmclock-tests dmclock-data-struct-tests)
+
+if(WITH_TESTS)
+  install(PROGRAMS
+    ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/dmclock-tests
+    ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/dmclock-data-struct-tests
+    DESTINATION bin)
+endif(WITH_TESTS)
```

Comment:
e.g. in `CMakeLists.txt` add a line that says `set(WITH_DMCLOCK_TESTS "enable the build of dmclock-tests and dmclock-data-struct-tests binaries` OFF)`

and then s/WITH_TESTS/WITH_DMCLOCK_TESTS/